android/build.gradle

```
@@ -207,3 +207,80 @@ dependencies {
     implementation "androidx.core:core-ktx:1.6.0"
     implementation "org.jetbrains.kotlin:kotlin-stdlib:$kotlin_version"
 }
+
+def isGHExampleApp() {
+    return project.hasProperty('isGHExampleApp') && project.property('isGHExampleApp') == "true"
+}
+
+def getAbsoluteCodegenArtifactsPaperDestination() {
+    if (!project.hasProperty('codegenArtifactsPaperDestination')) {
+        throw new Exception('[react-native-gesture-handler] Please fill codegenArtifactsPaperDestination variable in android/gradle.properties to point to the correct path to generated specs for paper')
+    }
+
+    return "${project.rootDir}/../../${project.property('codegenArtifactsPaperDestination')}"
+}
+
+def getAbsoluteCodegenArtifactsSource() {
+    if (!project.hasProperty('codegenArtifactsSource')) {
+        throw new Exception('[react-native-gesture-handler] Please fill codegenArtifactsSource variable in android/gradle.properties to point to the correct path to codegenerated artifacts')
```

Again, nitpicking here but I think these lines won't be touched for months.

```suggestion
def getAbsoluteCodegenArtifactsPaperDestination() {
    if (!project.hasProperty('codegenArtifactsPaperDestination')) {
        throw new Exception('"react-native-gesture-handler] Please fill codegenArtifactsPaperDestination variable in android/gradle.properties to point to the correct path to generated specs for paper.")
    }

    return "${project.rootDir}/../../${project.property('codegenArtifactsPaperDestination')}"
}

def getAbsoluteCodegenArtifactsSource() {
    if (!project.hasProperty('codegenArtifactsSource')) {
        throw new Exception("[react-native-gesture-handler] Please fill codegenArtifactsSource variable in android/gradle.properties to point to the correct path to codegenerated artifacts.")
```

---

Changed in https://github.com/software-mansion/react-native-gesture-handler/pull/2933/commits/7e4da8d6e5dcc10337a96a60d5e4c44c6d75283b